docs/examples/sync_job.py

```diff
@@ -0,0 +1,14 @@
+def sync_task(t):
```

Comment:
please include all imports required to run the example.

Reply:
Done.